src/cephadm/cephadm

```diff
@@ -2431,7 +2575,6 @@ class CephContainer:
 
     def run(self, timeout=DEFAULT_TIMEOUT):
         # type: (Optional[int]) -> str
-        logger.debug(self.run_cmd())
```

Comment:
We may want to keep that in?

Reply:
The command is logged in https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L765, so it's not necessary to do it twice.